wirebait.lua

```diff
@@ -1258,8 +1258,10 @@ function wirebait.buffer.new(data_as_hex_string, offset)
     if length <= 32 then
       local uint_val = self(byte_offset, byte_size):uint64();
       local bit_mask = tonumber(string.rep("1", length),2);
-      --return bwAnd(bwRshift(uint_val, right_bits_count), bit_mask);      
-      return (uint_val:rshift(right_bits_count)):band(bit_mask);
```

Comment:
@pffang thanks for your help!
That's a nice catch. I see what you are doing, I would just make a minor tweak to your change. The `UInt64` class already has a [`tonumber()`](https://github.com/MarkoPaul0/WireBait/blob/master/wirebait.lua#L411) method which we can use directly. So here instead of your change I would just modify the return statement like that:
```Lua
return uint_val:rshift(right_bits_count):band(bit_mask):tonumber()
```
or if you prefer:
```lua
local uint64_val = (uint_val:rshift(right_bits_count)):band(bit_mask);
return uint64_val:tonumber();
```
If you update your pull request with this change I will merge it in.

Reply:
@MarkoPaul0 Thank you for your suggestion.
I'm just a newbie in Lua language.